keyboards/latin60rgb/config.h

```diff
@@ -0,0 +1,89 @@
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+#pragma once
+
+/* USB Device descriptor parameter */
+#include "config_common.h"
+#define VENDOR_ID       0x6C63 // "lc" = latincompass
+#define PRODUCT_ID      0x6C69 // "lp" = latin60RGB
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    latincompass
+#define PRODUCT         Latin60RGB
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS {C7, C6, B6, B5, B4 }
+#define MATRIX_COL_PINS {F7, F6, F5, F4, F1, F0, E6, B0, B1, B2, B3, D6, D4, D3}
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION ROW2COL
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 3
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED true // turn off effects when suspended
+#    define RGB_MATRIX_KEYPRESSES
+#    define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#    define DISABLE_RGB_MATRIX_GRADIENT_UP_DOWN
+#    define DISABLE_RGB_MATRIX_BAND_SAT
+#    define DISABLE_RGB_MATRIX_BAND_PINWHEEL_SAT
+#    define DISABLE_RGB_MATRIX_BAND_SPIRAL_SAT
+#    define DISABLE_RGB_MATRIX_SOLID_REACTIVE_WIDE
+#    define DISABLE_RGB_MATRIX_SOLID_REACTIVE_MULTIWIDE
+#    define DISABLE_RGB_MATRIX_SOLID_REACTIVE_CROSS
+#    define DISABLE_RGB_MATRIX_SOLID_REACTIVE_MULTICROSS
+#    define DISABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+#    define DISABLE_RGB_MATRIX_SOLID_REACTIVE_MULTINEXUS
+#    define DISABLE_RGB_MATRIX_SPLASH
+#    define DISABLE_RGB_MATRIX_MULTISPLASH
+#    define DISABLE_RGB_MATRIX_SOLID_SPLASH
+#    define DISABLE_RGB_MATRIX_SOLID_MULTISPLASH
+#    define DISABLE_RGB_MATRIX_DIGITAL_RAIN
+#    define DRIVER_ADDR_1 0b1010000
+#    define DRIVER_ADDR_2 0b1010000 // this is here for compliancy reasons.
+#    define DRIVER_COUNT 2
+#    define DRIVER_1_LED_TOTAL 60
+#    define DRIVER_LED_TOTAL DRIVER_1_LED_TOTAL
+#endif
+#define RGB_DI_PIN B7
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 8
+#define RGBLIGHT_HUE_STEP 5
+#define RGBLIGHT_SAT_STEP 5
+#define RGBLIGHT_VAL_STEP 5
+#define RGBLIGHT_SLEEP
+#endif
```

Comment:
Does this even support RGB Light?   Because it's disabled in the rules.mk file for the keyboard.

Reply:
Ihave delete this,TAHNKS